src/Ship-AI.cpp

```diff
@@ -28,8 +28,8 @@ bool Ship::AITimeStep(float timeStep)
 
 		// just in case the AI left it on
 		ClearThrusterState();
-		for (int i = 0; i < Guns::GUNMOUNT_MAX; i++)
-			SetGunState(i, 0);
+		SetGunsFiringState(GunDir::GUN_REAR, 0);
```

Comment:
So we've lost the flexibility (_unused admittedly_) of having other gun mounts and types of gunmounts that aren't just front or backwards?

Reply:
see commets below